mkw_stats_bot/mkw_stats/utils/embed_builder.py

```
@@ -0,0 +1,180 @@
+"""
+Centralized Discord embed creation.
+
+Eliminates duplicated embed patterns across bot.py, commands.py, and handlers.
+"""
+
+import discord
+from typing import List, Dict, Optional
```

Ruff (0.15.0)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)